velox/core/SimpleFunctionMetadata.h

```diff
@@ -609,6 +629,27 @@ class SimpleFunctionMetadata : public ISimpleFunctionMetadata {
     signature_ = builder.build();
   }
 
+  template <typename T>
+  static std::string argumentToString(const std::vector<T>& arguments) {
```

Comment:
This looks very much like a copy-paste from helpMessage. Would it be possible to refactor to avoid copy-paste?